### PR TITLE
chore(1817) - Create a quick start guide

### LINKED ISSUE #1817: [Chore]: Create a quick start guide
## Is your chore related to a problem? Please describe:

- No user guide

## Describe the solution you'd like:

- Create a user guide

## Describe alternatives you've considered:

- None

### PR DESCRIPTION
**Pull request by hybridx**
## Closes / Fixes / Resolves

- Resolves #1817 

## Explain the feature/fix

- Added SPAship user guide documentaion

## Does this PR introduce a breaking change

- No

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

Screenshot 2022-07-22 at 8 27 01 PM

Screenshot 2022-07-22 at 8 27 44 PM

Screenshot 2022-07-22 at 8 27 59 PM

Screenshot 2022-07-22 at 8 28 08 PM

Screenshot 2022-07-22 at 8 28 21 PM

Screenshot 2022-07-22 at 8 28 34 PM

Screenshot 2022-07-22 at 8 28 54 PM

Screenshot 2022-07-22 at 8 29 06 PM

Screenshot 2022-07-22 at 8 29 15 PM

Screenshot 2022-07-22 at 8 29 28 PM

Screenshot 2022-07-22 at 8 29 43 PM

Screenshot 2022-07-22 at 8 30 00 PM

Screenshot 2022-07-22 at 8 30 09 PM

</details>

### CODE REVIEW

File: packages/documentation/docs/guide/user-guide/Quickstart.md [hunk 1]
```diff
@@ -1,5 +1,78 @@
...
+  - Manual Deployment: Update .spashiprc.yml. (Find more details in SPAship documentation) 
+
+
+#### Example
+```sh
+cd demo-spa
+
+npm install
+npm run build
+
```
Comment (soumyadip007) [inline packages/documentation/docs/guide/user-guide/Quickstart.md:73]: npx @spaship/cli@1.5.2 init

File: packages/documentation/docs/guide/user-guide/Quickstart.md [hunk 2]
```diff
@@ -1,5 +1,78 @@
...
+npx @spaship/cli@1.5.0 deploy --apikey=$KEY --env=https://deploy-url-from-spaship-manager.com --ref=1.0.0
```
Comment (soumyadip007) [inline packages/documentation/docs/guide/user-guide/Quickstart.md:74]: npx @spaship/cli@1.5.2 deploy --apikey=$KEY --env=https://deploy-url-from-spaship-manager.com --ref=1.0.0

File: packages/documentation/docs/guide/user-guide/cli.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+    ```sh
+    cd $HOME/projects/MyProject
+    spaship deploy --env prod MyProject-1.0.0.tgz
+    ```
+
+    #### Or you could also use the following complete cli
+    ```sh
+    cd demo-spa
+    npm install
+    npm run build
```
Comment (soumyadip007) [inline packages/documentation/docs/guide/user-guide/cli.md:247]: npm run build
npx @spaship/cli@1.5.0 init

File: packages/documentation/docs/guide/user-guide/cli.md [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+    npx @spaship/cli@1.5.0 deploy --apikey=70f19422-bf53-44b1-b664-f9b4636bea61 --env=https://deploy-url-from-spaship-manager.com --builddir=dist --ref=1.0.0
```
Comment (soumyadip007) [inline packages/documentation/docs/guide/user-guide/cli.md:248]: npx @spaship/cli@1.5.2 deploy --apikey=70f19422-bf53-44b1-b664-f9b4636bea61 --env=https://deploy-url-from-spaship-manager.com --builddir=dist --ref=1.0.0
Write Preview

File: packages/documentation/docs/guide/user-guide/Quickstart.md [hunk 5]
```diff
@@ -1,5 +1,78 @@
...
+```
+
```
Comment (soumyadip007) [inline packages/documentation/docs/guide/user-guide/Quickstart.md:76]: - Please check the [video tutorial](https://drive.google.com/file/d/150OyktZdmqMXKwNS1mDkqIo7ZvgyCCGp/view?usp=sharing)

**Review by soumyadip007 — CHANGES_REQUESTED** [review]

File: packages/documentation/docs/guide/user-guide/Quickstart.md (hunk 1 above)
Comment (hybridx) [reply packages/documentation/docs/guide/user-guide/Quickstart.md:73]: it should be rather @latest

Comment (hybridx) [reply packages/documentation/docs/guide/user-guide/Quickstart.md:73]: Good catch

File: packages/documentation/docs/guide/user-guide/Quickstart.md (hunk 5 above)
Comment (hybridx) [reply packages/documentation/docs/guide/user-guide/Quickstart.md:78]: Added

File: packages/documentation/docs/guide/user-guide/cli.md (hunk 3 above)
Comment (hybridx) [reply packages/documentation/docs/guide/user-guide/cli.md:247]: @latest added

File: packages/documentation/docs/guide/user-guide/Quickstart.md (hunk 2 above)
Comment (hybridx) [reply packages/documentation/docs/guide/user-guide/Quickstart.md]: Added @latest

**Review by soumyadip007 — APPROVED** [review]
Looks good to me.

**Comment by sayak-sarkar** [comment]
LGTM! 👍🏻

**Pull request merged**